tests/unit/api/objectGet.js

```diff
@@ -75,6 +75,155 @@ describe('objectGet API', () => {
         });
     });
 
+    const testPutBucketRequestObjectLock = {
+        bucketName,
+        namespace,
+        headers: {
+            host: `${bucketName}.s3.amazonaws.com`,
+            'x-amz-object-lock-enabled': true,
+        },
+        url: `/${bucketName}`,
+    };
+
+    const testPutObjectReqRetention = (date, mode) => new DummyRequest({
```

Comment:
Maybe a name that dictates it just creates a dummy request like, `createPutObjectDummyRequest` or `createPutDummy` or `createPutRequest`. I thought this was an actual test the first time I read it.